resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -500,7 +503,7 @@ latex --version
 You should see something like this if you were successful:
 
 ```
-pdfTeX 3.14159265-2.6-1.40.21 (TeX Live 2020)
+pdfTeX 3.14159265-2.6-1.40.25 (TeX Live 2023)
```

Comment:
Note that I'm having issues with the Latex PDF generation from jupyterlab. The WebPDF generation works fine and those PDFs look better than the regular PDF generation in my opinion. What do you think about scrapping regular PDF export in JupyterLab from our installation instructions? It would cut down an entire section (that's usually problematic) and WebPDF is better for quick export in my opinion. If someone really wants Latex they can run their notebook through quarto where the tinytex hopefully takes care of things more effectively than it does for jupyterlab (I haven't tried)

Reply:
@ttimbers I added the same tlmgr packages you did and it is working now, thank you! I think this is still something we might consider dropping in the future to simplify things. I don't find the Latex output from jupyter that helpful since it looks very different from the notebook and does not wrap code lines, so hopefully quarto will be a unified way to output scientific PDFs even if they are created as ipynb in the notebook interface (webpdf is still a useful quick alternative for non-scientific output)

Reply:
Great that it fixed this! For this year let's include this since we have not yet done a full move over to Quarto yet. That may take another year. But if that works out well then yes, we can likely remove this in future.

Reply:
@chendaniely - ping to you to move up Quarto CLI and add `pdfcol` and `soul` as Latex packages installed via `tlmgr` in Windows and to test these.